### ISSUE TITLE
Support for Real Devices?

### ISSUE THREAD

**Issue by winstondu** [uber/ios-snapshot-test-case#123]
I was playing around with this library, and I noticed that this library's functionality does not seem to work on real devices, but works perfectly fine on simulator.

Is there a method to allow my tests to support real devices?

In theory, it should be more than possible to snapshot a real device. [Reveal](https://revealapp.com/), for example, is a mac program that can interface with an iOS device and get its complete view hierarchy (as such, generating a screenshot is completely possible).

**Comment by BackdoorBANDIT** [comment]
I'm curious as well.  Works well on simulator, but fails on physical device (Error:  ...SnapshotError error 0).

**Comment by alanzeino** [comment]
I don't think we ever tried this because running on device is more friction than running on Simulator. If you have some potential fixes, feel free to PR them.
